oauth2/src/main/java/com/okta/spring/boot/oauth/config/OktaOAuth2Properties.java

```diff
@@ -71,6 +71,11 @@
      */
     private String postLogoutRedirectUri;
 
+    /**
+     * Identifies Opaque token
+     */
+    private Boolean opaque = Boolean.FALSE;
```

Comment:
Use small b boolean, otherwise, you need to deal with 3 states

Reply:
done